aamt_auth.pm

```diff
@@ -38,21 +38,29 @@ my  $REC_MY_NAME_VIRTUAL_HOST ='[NAME_VIRTUAL_HOST]' ;
 sub auth_main                                               
 {
     ui_clearScreen();         #used to clear screen defined in aamt_userinterface.pm file
-	ui_Title();               #used for title of the migartion kit defined in aamt_userinterface.pm file	
+    ui_Title() if(&utf_getRunMode() eq 'i');             #used for title of the migartion kit defined in aamt_userinterface.pm file
```

Comment:
use consistent coding style for blocks:
if (&utf_getRunMode() eq 'i')
{
    ui_Title();
}